openarc/openarc.c

```diff
@@ -1212,7 +1212,7 @@ arcf_list_load(struct conflist *list, char *path, char **err)
 	}
 
 	memset(buf, '\0', sizeof buf);
-	while (fgets(buf, sizeof buf - 1, f) != (char *) EOF)
+	while (fgets(buf, sizeof buf - 1, f) != NULL)
```

Comment:
Interesting: The man pages for fgets() on CentOS and on MacOS say it returns EOF (-1) when there's no more input, not NULL.  On BSD, it's NULL.  Should we check for both?

Reply:
I've looked on CentOS 7 and online on OSX and everywhere it says fgets() returns NULL in case of EOF or an error. Please double-check it.